portal-web/test/functional/com/liferay/portalweb/macros/DMFolder.macro

```diff
@@ -312,6 +312,9 @@ definition {
 		else if (isSet(copyToUnconnectedSite)) {
 			Alert.viewErrorMessage(errorMessage = "The item is not copied because the site and asset library are not connected.");
 		}
+		else if (isSet(copyToStagingSite)) {
+			Alert.viewErrorMessage(errorMessage = "Cannot copy folders to the live version of a group. Make your changes in staging and publish them to Live afterwards to make them public.");
```

Comment:
Hey @AliciaGarciaGarcia From an end user perspective I don't remember if we have any area in Portal that mentions "group" (usually it is "site"). I'm not sure if we are making that distinction at the user interface vs. code level. If we are we should probably update the literal here

Reply:
Hi @austinchiang 
Yeah, AFAIK for the Portal there is no"group", only "site" so if this message is shown, the message should be changed.